### ISSUE TITLE
Does prettier-eslint not honor disabling comments?

### ISSUE THREAD

**Issue by chrbala** [prettier/prettier-eslint#57]
I've been having an issue with prettier-eslint via prettier-atom. Basically, my eslint config uses dangling commas, but in some situations, dangling commas don't run correctly. For example, the below code won't work correctly when run in config files that aren't transpiled.

```javascript
someFunction(
  firstArg,
  secondArg,
)
```

prettier-eslint is adding back that comma when I have ``/* eslint-disable */`` at the top of my file. Does prettier-eslint not honor disabling of eslint features via comments?

**Comment by kentcdodds** [comment]
I documented in the README why this is happening. I hope that helps. Feel free to open a PR to change the defaults :)

I just realized that my update to the README doesn't really answer your question though. The fact of the matter is that prettier doesn't care about your eslint comments, so even though eslint doesn't run, prettier does. If you'd like to turn this issue into a feature request for that, we can discuss it. I think I'm in favor, but I'd like others to weigh in...
